src/NuGet.Core/NuGet.Common/Errors/WarningLevel.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace NuGet.Common
+{
+    /// <summary>
+    /// These are Warning Levels used by NuGet while throwing warnings.
+    /// These logically correspond to .NET spec at https://msdn.microsoft.com/en-us/library/13b90fz7(v=vs.140).aspx
+    /// 
+    /// We do not have a level 0 as that has no logical meaning of having a warning with level 0.
+    /// 
+    /// Severe - This should be used to throw warnings that just short of being an error.
+    ///          Can be ignored if the project warn level is 0.
```

Comment:
Should this be left out? We can't really set this ourselves. Same for the other settings on ignore